examples/aws/lambda_with_go_runtime/src/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-lambda-go/lambdacontext"
+)
+
+// The Lambda function will parse received events into this
+// data structure.
+type Event struct {
+	Name string `json:"name"`
+}
+
+type Response struct {
+	Status    string `json:"status"`
+	Message   string `json:"message"`
+	RequestId string `json:"request_id"`
+}
+
+func HandleRequest(ctx context.Context, event *Event) (Response, error) {
+	if event == nil {
+		return Response{}, fmt.Errorf("received nil event")
+	}
+
+	lctx, ok := lambdacontext.FromContext(ctx)
+	if !ok {
+		errorMessage := "could not retrieve lambda's context"
+		return Response{Status: "error", Message: errorMessage}, fmt.Errorf("%s", errorMessage)
+	}
+
+	// Logs directly to CloudWatch.
```

Comment:
QoL: not sure if the comment is needed

```suggestion

```

Reply:
I added it as a kind of courtesy to another developer using the example repository to start a Lambda project with Go, since it is not self-evident (in my opinion) that a log using the standard library `log` package will get redirected to CloudWatch. 

So for a developer just starting with Lambdas and Go, it might be a good pointer to know that he/she does not have to import any kind of AWS-specific package to log to CloudWatch.

Reply:
I understand your point. I just think that "logging to cloudwatch" is kind of wrong. Since cloudwatch rather checks the logs of the machine the lambda function runs on rather than the machine logging to cloudwatch :D

Totally fine if we just keep it 👍️